internal/super/flix.go

```diff
@@ -121,6 +129,7 @@ func execute(
 		}
 
 	case flixPath:
+		isLocal = true
 		file, err := os.ReadFile(flixQuery)
```

Comment:
```suggestion
		file, err := state.ReadFile(flixQuery)
```

Reply:
most other places in flow-cli where file reading/writing is done it is done on the ReaderWriter.

Reply:
made changes to use state to read and write files. Also dried up the code a bit.